app/error.tsx

```
@@ -0,0 +1,96 @@
+'use client'
+
+import Link from 'next/link'
+import { useEffect } from 'react'
+import { Home, RefreshCw, AlertTriangle } from 'lucide-react'
+import { Button } from '@/components/ui/button'
+import { Card, CardContent, CardDescription, CardHeader, CardTitle } from '@/components/ui/card'
+
+export default function Error({
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)